src/components/Banner.tsx

```diff
@@ -95,8 +95,8 @@ export const BannerComponent: ComponentConfig<BannerProps> = {
   fields: bannerFields,
   defaultProps: {
     text: {
-      fieldName: "",
-      staticValue: "Banner Text",
+      field: "",
+      constantValue: "Banner Text",
```

Comment:
Add `constantValueEnabled:true` to any of these default props that used to have fieldName: ""
That should make the constant value used by default in VE